requirements/edx/base.txt

```diff
@@ -97,7 +97,7 @@ docutils==0.15.2          # via botocore
 drf-yasg==1.16
 edx-ace==0.1.10
 edx-analytics-data-api-client==0.15.3
-edx-bulk-grades==0.5.11
+edx-bulk-grades==0.6.0
```

Comment:
Why do we need to touch bulk-grades in this change?

Reply:
`make upgrade`

Reply:
as it turns out it's irrelevant since alex got to prod as well

Reply:
was grabbed by `make upgrade` while #21613 was open